code/__DEFINES/footsteps.dm

```diff
@@ -28,6 +28,8 @@
 #define FOOTSTEP_MOB_RUST "footstep_rust"
 #define FOOTSTEP_OBJ_MACHINE "footstep_machine"
 #define FOOTSTEP_OBJ_ROBOT "footstep_robot"
+#define FOOTSTEP_OBJ_CYBORG_FLY "footstep_cyborg_fly"
```

Comment:
i think you forgot to stage/push the code where you actually add the `CYBORG_FLY` sounds. they're in your video but there doesn't appear to be any code for it in here.

Reply:
I wanted to add it at first but I decided on keeping it really simple for now, I'll remove the define.